styles.css

```diff
@@ -0,0 +1,116 @@
+* {
+  box-sizing: border-box;
+}
+.sr-only{
+  position: absolute;
+  left: -10000px;
+  top: auto;
+  width: 1px;
+  height: 1px;
+  overflow: hidden;
+}
+.grid-container {
+  display: grid;
+  grid-gap: 2px;
+  grid-template-columns: auto auto auto auto;
+  grid-template-rows: auto auto auto auto;
+  justify-content: space-evenly;
+}
+a:link, a:visited {
+  color: white;
+}
+a:link {
+  text-decoration: none;
+}
+.grid-container {
+  display: grid;
+  grid-gap: 2px;
+  grid-template-columns: auto auto auto auto;
+  grid-template-rows: auto auto auto auto;
+  justify-content: space-evenly;
+}
+.grid-item-header{
+  grid-column: 1 / 3;
+  grid-row: 1/4;
+  background: #03a3df;
+  padding: 10px;
+  background: #000000 url(images/header-image.jpg);
+}
+.header{
+  float: right;
+}
+.global-nav{
+  grid-column: 1 /3;
+  grid-row: 4/6;
+  background-color: lightgray;
+}
+.grid-item-blog {
+  grid-column: 1 /3;
+  grid-row: 4/8;
+}
+
+.grid-item-welcome{
+  grid-column: 2 /3;
+  grid-row: 6/8;
+  padding: 30px 30px 30px 5px
+}
+
+.dog{
+margin-top: 3px;
+margin-bottom: 3px;
+border-style: solid;
+border-width: thin;
+border-color: gray;
+}
+
+.dog-blog{
+  grid-column: 2 /3;
+  grid-row: 6/8;
+  padding: 30px 30px 30px 5px
+}
+
+.dog-photo{
+  width: 20rem;
+  height: 20rem;
+  float: left;
+}
+.welcome-image {
+  width: 20rem;
+  height: 20rem;
+  float: left;
+}
+.about-text {
+  float: right;
+}
+.grid-item-about{
```

Comment:
This isn't quite laying out your dogs in a grid. You'll want to rethink this so they match the design.